From: Mathieu Malaterre <malat@debian.org>
Date: Thu, 5 Jul 2012 00:00:01 +0000
Subject: Use standard debian location for config files

---
 dicom.ini.sqlite |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dicom.ini.sqlite b/dicom.ini.sqlite
index f750bee..ce5d09b 100644
--- a/dicom.ini.sqlite
+++ b/dicom.ini.sqlite
@@ -19,7 +19,7 @@ SOPClassList             = dgatesop.lst
 
 # Host for postgres or mysql only, name, username and password for database
 SQLHost                  = localhost
-SQLServer                = ./data/dbase/conquest.db3
+SQLServer                = /var/lib/conquest-dicom-server/dbase/conquest.sqlite
 Username                 = dontcare
 Password                 = dontcare
 PostGres                 = 0
@@ -83,4 +83,4 @@ OperatorConsole          = 127.0.0.1
 # Configuration of disk(s) to store images
 MAGDeviceThreshhold      = 0
 MAGDevices               = 1
-MAGDevice0               = ./data/
+MAGDevice0               = /var/lib/conquest-dicom-server/MAG0/
