From: Pablo Lorenzzoni <spectra@debian.org>
Date: Mon, 27 Jan 2014 00:00:01 +0000
Subject: List configfiles with path in dicom.ini.sqlite

---
 dicom.ini.sqlite | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/dicom.ini.sqlite b/dicom.ini.sqlite
index 896c182..ae4f7da 100644
--- a/dicom.ini.sqlite
+++ b/dicom.ini.sqlite
@@ -13,9 +13,10 @@ MyACRNema                = CONQUESTSRV1
 TCPPort                  = 11112
 
 # Reference to other files: known dicom servers; database layout; sops
-ACRNemaMap               = acrnema.map
-kFactorFile              = dicom.sql
-SOPClassList             = dgatesop.lst
+ACRNemaMap               = /etc/conquest-dicom-server/acrnema.map
+kFactorFile              = /etc/conquest-dicom-server/dicom.sql
+SOPClassList             = /etc/conquest-dicom-server/dgatesop.lst
+Dictionary               = /etc/conquest-dicom-server/dgate.dic
 
 # Host for postgres or mysql only, name, username and password for database
 SQLHost                  = localhost
