From: Pablo Lorenzzoni <spectra@debian.org>
Date: Thu, 6 Mar 2014 15:34:35 -0300
Subject: Fix mismatched new/delete in dbsql.cpp

---
 dbsql.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dbsql.cpp b/dbsql.cpp
index 5240a40..e31249a 100644
--- a/dbsql.cpp
+++ b/dbsql.cpp
@@ -2309,7 +2309,7 @@ MakeSafeStringValues (
 	(*sout) = '\'';++sout;
 	(*sout) = '\0';
 	
-	delete s;
+	delete [] s;
 	return ( TRUE );
 	}
 
