From: Pablo Lorenzzoni <spectra@debian.org>
Date: Thu, 6 Mar 2014 17:02:42 -0300
Subject: Fix mismatched new/delete in buffer.cxx

---
 buffer.cxx |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/buffer.cxx b/buffer.cxx
index eeb4d69..b6b1842 100644
--- a/buffer.cxx
+++ b/buffer.cxx
@@ -102,7 +102,7 @@ BufferSpace	::	~BufferSpace()
 	{
 	if ( !isTemp )
 		if ( Data )
-			delete Data;
+			delete [] Data;
 	}
 
 Buffer	::	Buffer()
