From: Pablo Lorenzzoni <spectra@debian.org>
Date: Thu, 6 Mar 2014 14:54:52 -0300
Subject: Fix cppcheck bug 6 - Reviewed by Marcel

---
 dgate.cpp |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/dgate.cpp b/dgate.cpp
index 6a8c109..9fbd879 100644
--- a/dgate.cpp
+++ b/dgate.cpp
@@ -5950,11 +5950,16 @@ extern "C"
   { unsigned int i, N, flds=1;
     const char *items[4];
     SQLLEN SQLResultLength;
+
+    // For reference: https://github.com/spectra/conquest-dicom-server/commit/2c0077af8638682e93787897a3316a2bbba3c7e7#commitcomment-5388219
+    // items[0] is the database name
+    // items[1] is the fields
+    // items[2] is the (optional) query and
+    // items[3] is the optional sort order
     for (i=0; i<4; i++) items[i]=lua_tostring(L,i+1);
     if (items[1]) for (i=0; i<strlen(items[1]); i++) if (items[1][i]==',') flds++;
     if (items[2]) if (*items[2]==0) items[2]=NULL;
     if (items[3]) if (*items[3]==0) items[3]=NULL;
-    if (items[4]) if (*items[4]==0) items[4]=NULL;
 
     Database DB;
     if (DB.Open ( DataSource, UserName, Password, DataHost ) )
