From: Pablo Lorenzzoni <spectra@debian.org>
Date: Thu, 6 Mar 2014 15:43:54 -0300
Subject: Fix mismatched new/delete in device.cpp

---
 device.cpp |   20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/device.cpp b/device.cpp
index 0d4458f..d2f7eaa 100644
--- a/device.cpp
+++ b/device.cpp
@@ -185,12 +185,12 @@ void FreeDeviceTables(void)
 { int i;
 
   for (i=0; i<(int)MAGDevices; i++)
-    delete MAGDeviceTable[i];
-  delete MAGDeviceTable;
+    delete [] MAGDeviceTable[i];
+  delete [] MAGDeviceTable;
 
   for (i=0; i<(int)MIRRORDevices; i++)
-    delete MIRRORDeviceTable[i];
-  delete MIRRORDeviceTable;
+    delete [] MIRRORDeviceTable[i];
+  delete [] MIRRORDeviceTable;
 
   for (i=0; i<(int)MOPDevices; i++)
     delete MOPDeviceTable[i];
@@ -201,16 +201,16 @@ void FreeDeviceTables(void)
   delete CDRDeviceTable;
 
   for (i=0; i<(int)CACHEDevices; i++)
-    delete CACHEDeviceTable[i];
-  delete CACHEDeviceTable;
+    delete [] CACHEDeviceTable[i];
+  delete [] CACHEDeviceTable;
 
   for (i=0; i<(int)JUKEBOXDevices; i++)
-    delete JUKEBOXDeviceTable[i];
-  delete JUKEBOXDeviceTable;
+    delete [] JUKEBOXDeviceTable[i];
+  delete [] JUKEBOXDeviceTable;
 
-  delete MAGDeviceSpace;
+  delete [] MAGDeviceSpace;
 
-  delete MIRRORDeviceSpace;
+  delete [] MIRRORDeviceSpace;
 }
 
 
