Description: Use IANA port for the daemon
Author: Pablo Lorenzzoni <spectra@debian.org>
Forwarded: not-needed

--- conquestlinux1417c.orig/dicom.ini.sqlite
+++ conquestlinux1417c/dicom.ini.sqlite
@@ -10,7 +10,7 @@
 
 # Network configuration: server name and TCP/IP port#
 MyACRNema                = CONQUESTSRV1
-TCPPort                  = 5678
+TCPPort                  = 11112
 
 # Reference to other files: known dicom servers; database layout; sops
 ACRNemaMap               = acrnema.map
